Source/WebCore/page/DragController.cpp

```diff
@@ -1190,14 +1190,14 @@ bool DragController::startDrag(LocalFrame& src, const DragState& state, OptionSe
 
         client().willPerformDragSourceAction(DragSourceAction::Link, dragOrigin, dataTransfer);
         if (!dragImage) {
-            TextIndicatorData textIndicator;
-            dragImage = DragImage { createDragImageForLink(element, linkURL, textContentWithSimplifiedWhiteSpace, textIndicator, m_page->deviceScaleFactor()) };
+            auto [dragImageRef, textIndicator] = createDragImageForLink(element, linkURL, textContentWithSimplifiedWhiteSpace, m_page->deviceScaleFactor());
+            dragImage = DragImage { dragImageRef };
```

Comment:
While `DragImage` currently does not support move semantics, it doesn't hurt to:
```suggestion
            dragImage = DragImage { WTFMove(dragImageRef) };
```

Right now the `WTFMove(...)` will be a no-op, but once we have this constructor, this will move the text indicator:

```cpp
DragImage(RefPtr<TextIndicator>&&, std::optional<Path>&&);
```